packages/rulesets/generated/spectral/az-dataplane.js

```diff
@@ -1942,6 +1950,8 @@ const ruleset = {
     rules: {
         AdditionalPropertiesAndProperties: {
             description: "Don't specify additionalProperties as a sibling of properties.",
+            disableForTypeSpec: true,
+            disableForTypeSpecReason: "TODO",
```

Comment:
```suggestion
            disableForTypeSpecReason: "Covered by TSP's '@azure-tools/typespec-azure-core/bad-record-type' rule.",
```

I believe this is the same rule